cli/tests/test_windows_release_claims.py

```
@@ -19,15 +19,14 @@
 def test_windows_release_metadata_is_exact() -> None:
-    assert WINDOWS_SUPPORTED_CONNECTORS == {"codex", "claudecode", "amp"}
+    assert WINDOWS_SUPPORTED_CONNECTORS == {"codex", "claudecode", "amp", "hermes"}
     assert WINDOWS_NOT_CERTIFIED_CONNECTORS == {
         "cursor",
         "windsurf",
         "geminicli",
         "copilot",
         "antigravity",
         "opencode",
-        "hermes",
     }
```

Ruff (0.16.2)

[warning] Yoda condition detected

(SIM300)